src/App.tsx

```diff
@@ -1,14 +1,23 @@
+import { useState } from 'react';
+
 import { AboutMeComponent } from '@/components/about/AboutMe.component.tsx';
 import { FooterComponent } from '@/components/footer/Footer.component.tsx';
 import { HeaderComponent } from '@/components/header/Header.component.tsx';
+import { ProductsList } from '@/components/ProductsList/ProductsList.component.tsx';
 
 import './App.css';
 
 function App() {
+    const ABOUT_ME: string = 'AboutMe';
+    const PRODUCT_LIST: string = `ProductList`;
+
+    const [activePage, setActivePage] = useState('AboutMe');
```

Comment:
add generic type `Pages`

Reply:
@serhii-tsivinskyi-moc I'm not sure I did the right thing